wallet/build.gradle

```diff
@@ -220,6 +221,15 @@ android {
         }
     }
 
+    signingConfigs {
+        debug {
+            storeFile file('keystore/debug.keystore')
+            storePassword "android"
+            keyAlias "androiddebugkey"
+            keyPassword "android"
+        }
+    }
+
```

Comment:
Didn't we remove the leaked debug key recently?

Reply:
Yes, thanks for catching this.  I needed to add this back for working with an install that was built using the old debug key.  It will be removed before merging.